test/functional/apps/console/_console.js

```diff
@@ -36,37 +36,31 @@ export default function ({ getService, getPageObjects }) {
   const PageObjects = getPageObjects(['common', 'console']);
 
   describe('console app', function describeIndexTests() {
-    before(function () {
+    before(async function () {
       log.debug('navigateTo console');
-      return PageObjects.common.navigateToApp('console');
+      await PageObjects.common.navigateToApp('console');
     });
 
-    it('should show the default request', function () {
+    it('should show the default request', async function () {
       // collapse the help pane because we only get the VISIBLE TEXT, not the part that is scrolled
-      return PageObjects.console.collapseHelp()
-        .then(function () {
-          return retry.try(function () {
-            return PageObjects.console.getRequest()
-              .then(function (actualRequest) {
-                expect(actualRequest.trim()).to.eql(DEFAULT_REQUEST);
-              });
-          });
-        });
+      await PageObjects.console.collapseHelp();
+      let  actualRequest;
+      await retry.try(async function () {
+        actualRequest = await PageObjects.console.getRequest();
+        log.debug(actualRequest);
+      });
+      expect(actualRequest.trim()).to.eql(DEFAULT_REQUEST);
     });
 
-    it('default request response should include `"timed_out": false`', function () {
+    it('default request response should include `"timed_out": false`', async function () {
       const expectedResponseContains = '"timed_out": false,';
-
-      return PageObjects.console.clickPlay()
-        .then(function () {
-          return retry.try(function () {
-            return PageObjects.console.getResponse()
-              .then(function (actualResponse) {
-                log.debug(actualResponse);
-                expect(actualResponse).to.contain(expectedResponseContains);
-              });
-          });
-        });
+      await PageObjects.console.clickPlay();
+      let  actualResponse;
+      await retry.try(async function () {
+        actualResponse = await PageObjects.console.getResponse();
```

Comment:
You should probably remove the `let actualResponse;` above and change this line to `const actualResponse = ` since it's only being set once and used right after that.